assets/css/NewEvaluation.less

```diff
@@ -0,0 +1,148 @@
+body {
+box-sizing: border-box;
+background-color: #f7f7f7;
+
+}
+
+.borderSwitch{
+border-bottom: solid thick #03a9f4;
+text-align: center;
+}
+
+.pgSwitch{
+text-decoration: none;
+color:black;
+font-size: 170%;
+margin-left: 5rem;
+padding-left: 2rem;
+padding-right: 2rem;
+}
+
+.homeLogo {
+margin-right: 9rem;
+margin-left: 2rem;
+}
+
+.header-img {
+width: 200px;
+}
+
+.dataForm {
+width: 100%;
+text-align: left;
+padding: 15px;
+margin-bottom: 25px;
+
+input{
+    text-align: left;
+    width: 32%;
+    min-height: 25px;
+    box-shadow: 0px 1px -15px -10px black inset;
+    border-bottom: 1px solid #ccc;         
+}
+}
+
+.table{
+width: 100%;
+height: 5rem;
+border-collapse: collapse;
+text-align: center;
+
+.row1{
```

Comment:
Sensei, i will solve the mess in CSS class-names and random curly braces in the text until next week.